backend/ecommerce/schema.py

```diff
@@ -143,6 +143,28 @@ def mutate(self, info, name, email, phone, password):
         )
 
 
+class AddReview(graphene.Mutation):
+    id = graphene.String()
+    rating = graphene.Int()
+    text = graphene.String()
+
+    class Arguments:
+        rating = graphene.Int()
+        text = graphene.String()
+        productId = graphene.String()
+
+    @login_required
+    def mutate(self, info, rating, productId, **kwargs):
+        text = kwargs.get('text', None)
```

Comment:
Why not take it as a param?

Reply:
If we take it as param then it becomes required